src/Athens-Examples/AthensFlakeDemo.class.st

```diff
@@ -14,6 +14,12 @@ Class {
 	#category : #'Athens-Examples-Demos'
 }
 
+{ #category : #'instance creation' }
+AthensFlakeDemo class >> open [
+	<script>
```

Comment:
The example pragma is a better fit when something is actually opened, not just run as a script